soccer/src/soccer/strategy/agent/position/position.hpp

```diff
@@ -75,6 +76,9 @@ class Position {
     void update_alive_robots(std::vector<u_int8_t> alive_robots);
     const std::string get_name();
 
+    //returns the current state of the robot
+    virtual std::string return_current_state() = 0;
```

Comment:
```suggestion
    virtual std::string get_current_state() = 0;
```
Just a quick suggestion. It's not common to name a function "return...". I would suggest either `get_current_state` or even simply `current_state`